tests/test_clients.py

```diff
@@ -18,3 +18,6 @@ def test_alter_fullscreen(hlwm):
     assert hlwm.get_attr('clients.focus.fullscreen') == 'true'
     hlwm.call('attr clients.focus.fullscreen toggle')
     assert hlwm.get_attr('clients.focus.fullscreen') == 'false'
+
+def test_fullscreen_completion(hlwm):
+    assert hlwm.complete("fullscreen") == 'false off on toggle true'.split(' ')
```

Comment:
It feels a bit hacky to me, to use `split` like that. If most completion tests are going to look like that, I'd propose a different approach. If we were using pyhamcrest, we could introduce a custom matcher here. Within out toolset, we could make `hlwm.complete()` return an object offers a matcher method that takes care of the string splitting.

Reply:
I'm really unsure about the future of tests for completions. I don't want to copy the entire completions to the test suite https://github.com/herbstluftwm/herbstluftwm/blob/master/src/command.cpp#L31

Reply:
It's a very large surface to cover. Maybe we could try a heuristic approach?

 * for every command dispatched in the test suite, verify that it would have been "reachable" via 
completions
 * enumerate and run everything that is offered via completions, expect it to succeed (not sure if this is feasible)